examples/getting-started-session-based/01-ETL-with-NVTabular.ipynb

```diff
@@ -1,1059 +1,932 @@
 {
```

Comment:
<p>Same - we compute the workflow twice (fit transform) - would it not better to store it to disk and then read it to split it into days?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>